frontend/playground-frontend/src/pages/feedback.tsx

```diff
@@ -9,8 +9,8 @@ import Spacer from "@/common/components/Spacer";
 import ReCAPTCHA from "react-google-recaptcha";
 
 import { COLORS, GENERAL_STYLES } from "../constants";
-//import { toast } from "react-toastify";
-//import { sendToBackend } from "../helper_functions/TalkWithBackend";
+import { toast } from "react-toastify";
+import { sendToBackend } from "../components_old/helper_functions/TalkWithBackend";
```

Comment:
Use RTK queries. We're deprecating `TalkWithBackend.js`

Reply:
^^ Anything in components_old is deprecated, you can find examples of RTK queries in any ts file with the word api in it.